### ISSUE TITLE
Move settings to their own sub-menu

### ISSUE THREAD

**Issue by dartiss** [dartiss/draft-concluder#17]
I think I have enough settings to justify this.

Will also be able to add in help details, etc., which will be advantageous.

**Comment by jphorn** [comment]
Yes please. The General section of WordPress admin is already pretty crowded, so its own subsection would be much better.

**Comment by slaFFik** [comment]
I vote for "Tools > Draft Concluder" :)

**Comment by jphorn** [comment]
I'd rather have it under Settings > Draft Concluder where all the plugin settings are.
